irm/README.rst

```diff
@@ -7,9 +7,9 @@ Python Client for Stackdriver Incident Response & Management API (`Alpha`_)
 - `Product Documentation`_
 
 .. _Alpha: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/README.rst
-.. _Stackdriver Incident Response & Management API: https://cloud.google.com/irm
-.. _Client Library Documentation: https://googlecloudplatform.github.io/google-cloud-python/latest/irm/usage.html
-.. _Product Documentation:  https://cloud.google.com/irm
+.. _Stackdriver Incident Response & Management API: https://cloud.google.com/incident-response/
+.. _Client Library Documentation: https://googlecloudplatform.github.io/google-cloud-python/latest/irm
```

Comment:
Might as well update all the `GoogleCloudPlatform` / `googlecloudplatform` bits to `googleapis` while we're at it.  And we should fix the template which generates them, too.

Reply:
Updated, and opened a PR to fix the template: https://github.com/googleapis/gapic-generator/pull/2582